mocks/mock-data/disputes.ts

```diff
@@ -48,6 +48,207 @@ const VISA_CONSUMER_DEFENSE_REASONS = ['InvalidChargeback', 'MerchandiseReceived
 
 const VISA_FRAUD_DEFENSE_REASONS = ['AdditionalInformation'] as const;
 
+const DEFAULT_DETAIL_DEFENSE: IDisputeDetail['defense'] = {
+    defendedOn: getDate(-1),
+    defendedThroughComponent: true,
+    reason: 'Defense reason',
+    suppliedDocuments: ['GoodsOrServicesProvided', 'WrittenRebuttal'],
+};
+const DEFAULT_DETAIL_PAYMENT: IDisputeDetail['payment'] = {
+    balanceAccount: { timeZone: 'UTC', description: 'Main balance account' },
+    isRefunded: false,
+    merchantReference: 'b2c3d4e5-f6g7-5890-efgh',
+    paymentMethod: { lastFourDigits: '1234', type: 'visa' },
+    pspReference: 'a1b2c3d4-e5f6-4789-abcd',
+};
+
+const DEFAULT_DETAIL_DISPUTE: IDisputeDetail['dispute'] = {
+    amount: {
+        value: 211100,
+        currency: 'EUR',
+    },
+    dueDate: new Date(new Date().setHours(23)).toISOString(),
+    createdAt: getDate(-10),
+    pspReference: 'a1b2c3d4-e5f6-4789-abcd-000000000001',
+    reason: {
+        category: 'REQUEST_FOR_INFORMATION',
+        title: 'Request for information',
+        code: '1234',
+    },
+    allowedDefenseReasons: [],
+    defensibility: 'ACCEPTABLE',
+    type: 'CHARGEBACK',
+    status: 'UNDEFENDED',
+};
+
+const DEFAULT_DISPUTE_DETAIL: IDisputeDetail = {
+    dispute: DEFAULT_DETAIL_DISPUTE,
+    payment: DEFAULT_DETAIL_PAYMENT,
+};
+
+// CHARGEBACKS
+
+export const CHARGEBACK_UNDEFENDED: IDisputeDetail = {
+    dispute: {
+        ...DEFAULT_DISPUTE_DETAIL.dispute,
+        status: 'UNDEFENDED',
+    },
+    payment: DEFAULT_DETAIL_PAYMENT,
+};
+
+export const CHARGEBACK_WON: IDisputeDetail = {
+    ...DEFAULT_DISPUTE_DETAIL,
+    dispute: {
+        ...DEFAULT_DISPUTE_DETAIL.dispute,
+        status: 'WON',
+        defensibility: 'NOT_ACTIONABLE',
+    },
+    defense: DEFAULT_DETAIL_DEFENSE,
+};
+
+export const CHARGEBACK_LOST: IDisputeDetail = {
+    ...DEFAULT_DISPUTE_DETAIL,
+    dispute: {
+        ...DEFAULT_DISPUTE_DETAIL.dispute,
+        status: 'LOST',
+        defensibility: 'NOT_ACTIONABLE',
+    },
+    defense: DEFAULT_DETAIL_DEFENSE,
+};
+
+export const CHARGEBACK_ACCEPTED: IDisputeDetail = {
+    ...DEFAULT_DISPUTE_DETAIL,
+    dispute: {
+        ...DEFAULT_DISPUTE_DETAIL.dispute,
+        status: 'ACCEPTED',
+        defensibility: 'NOT_ACTIONABLE',
+    },
+};
```

Comment:
This implicitly has `defense` field, which is not applicable for the `ACCEPTED` case. Consider this:

```suggestion
export const CHARGEBACK_ACCEPTED: IDisputeDetail = {
    dispute: {
        ...DEFAULT_DETAIL_DISPUTE,
        defensibility: 'NOT_ACTIONABLE',
        status: 'ACCEPTED',
    },
    payment: DEFAULT_DETAIL_PAYMENT,
};
```